dojson/contrib/marc21/schemas/marc21/bibliographic/bdleader.json

```diff
@@ -0,0 +1,9 @@
+{
+    "type": "object",
+    "properties": {
+        "leader": {
+            "description": "Leader",
+            "type": "string"
```

Comment:
But you have added https://github.com/inveniosoftware/dojson/pull/133/files#diff-3e9ef40633bd368357481cc88a142ee0R97